src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countOfOperations = 0;
+
+  return x => {
+    countOfOperations++;
+
+    return (y) => {
+      if (countOfOperations >= 3 && countOfOperations % 2 === 0) {
```

Comment:
```suggestion
      if (countOfOperations > 3 && countOfOperations % 2 === 0) {
```
it should be greater than 3, not greater or equal